ideascube/library/views.py

```diff
@@ -13,10 +13,10 @@
 from django.views.generic import (CreateView, DeleteView, DetailView, FormView,
                                   ListView, UpdateView, View)
 
+from ideascube.configuration.models import Configuration
```

Comment:
The API was completely overhauled in the latest push to this branch, in big part following what you suggested here.

Thanks!